### ISSUE TITLE
Add aria-label and data-action attributes

### ISSUE THREAD

**Issue by bennymeier** [ailon/cropro#18]
Hi,

I would like to add my own language to the `title`- and `aria-label`-attributes.
The top and the bottom toolbar are missing the `aria-label`-attributes completly and in `Marker.js` you added also `data-action`-attributes where I can explicitly get the button which I want.

Currently:
```html
<div title="Aspect ratio" class="__cropro__0_toolbar_button cropro__toolbar__button"></div>
```

Could be:
```html
<div aria-label="Aspect ratio" title="Aspect ratio" data-action="ratio" class="__cropro__0_toolbar_button cropro__toolbar__button"></div>
```

Could you please add this too for `Cropro`? That would be very nice!

Thank you.
